pkg/machine/qemu/machine.go

```diff
@@ -630,18 +644,21 @@ func (v *MachineVM) Start(name string, _ machine.StartOptions) error {
 	}
 
 	v.waitAPIAndPrintInfo(forwardState, forwardSock)
-
+	//	v.Starting = false
```

Comment:
What is this?

Reply:
leftover as well thanks for catching that. I Opted for the `defer` instead.